prefect_gcp/credentials.py

```diff
@@ -72,6 +72,9 @@ class GcpCredentials:
         project: Name of the project to use.
     """
 
+    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/4CD4wwbiIKPkZDt4U3TEuW/c112fe85653da054b6d5334ef662bec4/gcp.png?h=250"  # noqa
+    _block_type_name = "Google Cloud Platform Credentials"
+
```

Comment:
For display in the UI
```suggestion
    _block_type_name="GCP Credentials"
    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/4CD4wwbiIKPkZDt4U3TEuW/c112fe85653da054b6d5334ef662bec4/gcp.png?h=250"  # noqa
    _block_type_name = "Google Cloud Platform Credentials"

```

Reply:
Slightly confused about this; block_type_name is defined twice

Reply:
Ah, I missed that! My preference would be to name the block "GCP Credentials" so that it's not too long for the UI.